src/commands/restore.rs

```diff
@@ -113,26 +113,18 @@ pub async fn restore<'x>(cli: &mut Connection, options: &Options,
     }
 }
 
-async fn restore_db<'x>(cli: &mut Connection, options: &Options,
+async fn restore_db<'x>(cli: &mut Connection, _options: &Options,
     params: &RestoreCmd)
     -> Result<(), anyhow::Error>
 {
     use PacketType::*;
     let RestoreCmd {
-        allow_non_empty, path: ref filename,
+        path: ref filename,
         all: _, verbose: _,
     } = *params;
-    if !allow_non_empty {
-        if is_empty_db(cli).await.context("Error checking if the DB is empty")? {
-            if options.command_line {
-                return Err(anyhow::anyhow!("\
-                    cannot restore: the database is not empty; \
-                    consider using the --allow-non-empty option"));
-            } else {
-                return Err(anyhow::anyhow!(
-                    "cannot restore: the database is not empty"));
-            }
-        }
+    if is_empty_db(cli).await.context("Error checking if the DB is empty")? {
```

Comment:
if !is_empty?

Reply:
Well, yes, misleading the name of the function. Fixed.